plugins/cue_points/base/lib/kCuePointManager.php

```diff
@@ -398,6 +444,23 @@ public function shouldConsumeChangedEvent(BaseObject $object, array $modifiedCol
 			return true;
 		}
 
+		if ( self::wasEntryClipped($object, $modifiedColumns) ) {
+			return true;
+		}
+
+		return false;
+	}
+
+	public static function wasEntryClipped(BaseObject $object, array $modifiedColumns)
+	{
+		if ( ($object instanceof entry)
+			&& in_array(entryPeer::CUSTOM_DATA, $modifiedColumns)
```

Comment:
No need to check entryPeer::CUSTOM_DATA if you already check isCustomDataModified.